### PR TITLE
feat(generative-ai): update generative ai input to be resizable text area COMPASS-7940

### PR DESCRIPTION
**Pull request by Anemy**
COMPASS-7940

This makes it so that multi-line input can go in the prompt input for query and aggregation generation.

https://github.com/mongodb-js/compass/assets/1791149/9b480190-2c50-4366-b068-a1af9e93d5f1

### CODE REVIEW

File: packages/compass-generative-ai/src/components/generative-ai-input.tsx [hunk 1]
```diff
@@ -341,24 +397,38 @@ function GenerativeAIInput({
       <div className={inputBarContainerStyles}>
         <div className={inputContainerStyles}>
           <div className={isFetching ? gradientAnimationStyles : undefined}>
-            <div
-              className={contentWrapperStyles}
-              data-testid="ai-user-text-input-wrapper"
-            >
-              <TextInput
+            <div className={contentWrapperStyles}>
+              <button
```
Comment (Anemy) [inline packages/compass-generative-ai/src/components/generative-ai-input.tsx:401]: drive-by: moved this button up so that it's earlier in the tab order which is more inline with where it's rendered.

File: packages/compass-query-bar/src/components/query-bar.tsx [hunk 2]
```diff
@@ -218,14 +214,12 @@ export const QueryBar: React.FunctionComponent<QueryBarProps> = ({
...
       {isAIFeatureEnabled && (
-        <div className={queryAIContainerStyles}>
-          <QueryAI
-            onClose={() => {
-              onHideAIInputClick?.();
-            }}
-            show={isAIInputVisible}
-          />
-        </div>
+        <QueryAI
```
Comment (Anemy) [inline packages/compass-query-bar/src/components/query-bar.tsx:217]: drive-by: align the query and aggregation ai prompt input margins.

**Review by mcasimir — APPROVED** [review]
